ArthurAtangana/club-management-system
# feat: implement trainer schedule management use case

## What type of PR is this? (check all applicable)


## Description
Trainers can manage their schedule by adding or removing availability.

Would be nice to tell the user why an update fails to happen.
Reasons include:
- Already available that day
- Have a member registered in a class for that day

## How to test
- Login as Jane Doe (userId=2) and enter command to manage schedule.
- Add an availability. 
- If availability updates, add again to confirm that it fails to update the second time.
- Remove the availability you just added.

## Added/updated tests?

- [X] No, and this is why: Need a batch UI to actually write tests for the views.

## Screenshots

### Adding availability
<img width="633" alt="image" src="https://github.com/ArthurAtangana/club-management-system/assets/77079888/27b0a3bd-ea98-4490-b3a9-ed6fc8d058ae">
